### DISCUSSION TITLE
Tracing: filter spans outside of the selected timerange

### DISCUSSION THREAD

**Discussion by pracucci** [grafana/grafana#66484] [Explore]
The tracing view in the Explore section allows you to select a timerange:

![Screenshot 2023-04-13 at 17 19 49](https://user-images.githubusercontent.com/1701904/231806872-3ce49d56-ebce-4038-9ca7-af37755cf867.png)

When a custom timerange is selected, the view still showes all spans, so it looks like it's behaving like a "zoom" for the right side panel (the one showing the colored bars) rather than a "filter":

![Screenshot 2023-04-13 at 17 20 41](https://user-images.githubusercontent.com/1701904/231807404-6c0cdc8b-2f97-4717-b9bb-f7032f2dfded.png)

**Proposal**: I propose to filter out spans whose timing is outside the selected timerange.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82719. Feel free to continue the discussion around this there. Thank you!
